src/employees.js

```diff
@@ -101,6 +101,16 @@ const formData = [
         required: true,
       },
       { name: "taxId", type: "string", label: "Tax Number" },
+      { name: "gender",
+        type: "selectList",
+        source: "gender",
+        choices: [
+          { id: 'Male', name: 'Male' , label: 'Male' },
+          { id: 'Female', name: 'Female' , label: 'Female' },
+          { id: 'NO_BINARY', name: 'NO_BINARY', label: 'NO_BINARY' }
```

Comment:
Please check this comment: https://github.com/entropy-code/entropay-employees/pull/241#discussion_r1704026246
It will likely impact this.